packages/ckeditor5-build-helpjuice-v2/src/plugins/tab/tabediting.js

```diff
@@ -84,10 +84,16 @@ export default class TabEditing extends Plugin {
 		conversion.for("editingDowncast").elementToElement({
 			model: "tab",
 			view: (modelElement, { writer: viewWriter }) => {
+				const link = viewWriter.createAttributeElement("a", {
+					class: 'helpjuice-tab-info',
+					href: 'https://help.helpjuice.com/how-to-insert-tabs',
```

Comment:
I noticed that this article on HJ is on the wrong category:

![image](https://github.com/Helpjuice/ckeditor5/assets/28270050/c4c414b7-c1c4-490e-a4e0-410cd9837557)

Can you update the article's category?

Reply:
Sure, I'll make sure to update it. 
Also, I will update the content of all articles before deploying.